README.md

```diff
@@ -56,16 +55,15 @@ export default config({
 ### Advanced usage
 
 ```js
-import config from 'ca-webpack-config';
-
-export default config().merge({
-  module: {
-    loaders: [
-      // Append a loader
-    ],
-  }
+import webpack from 'webpack';
+import Config from 'webpack-config';
+
+export default new Config().extend(
+  'config-ca',
+  'config-ca/eslint',
+).merge({
```

Comment:
extending both `ca` and `ca/eslint` is unnecessary as `ca` already includes `ca/eslint`. Change to:
```
export default new Config().extend(
  'ca/base',
  'ca/eslint',
).merge({ ...
```